samples/common/src/main/java/com/bumble/appyx/samples/common/profile/ProfileCardNode.kt

```diff
@@ -24,48 +24,44 @@ import com.bumble.appyx.core.node.Node
 
 class ProfileCardNode(
     buildContext: BuildContext,
-    private val imageUrl: String,
-    private val name: String,
-    private val age: Int
+    private val profile: Profile
 ) : Node(buildContext) {
 
     @Composable
     @Override
     override fun View(modifier: Modifier) {
-        ProfileCard(imageUrl, name, age, modifier)
+        ProfileCard(profile)
     }
 }
 
 @Composable
 fun ProfileCard(
-    imageUrl: String,
-    name: String,
-    age: Int,
+    profile: Profile,
     modifier: Modifier = Modifier
 ) {
     BoxWithConstraints(modifier = modifier) {
         if (maxHeight < threshold) {
-            ProfileImage(imageUrl)
+            ProfileImage(profile.drawable)
         } else {
-            ProfileImage(imageUrl = imageUrl)
+            ProfileImage(profile.drawable)
             Column(
                 modifier = Modifier
                     .align(Alignment.BottomStart)
-                    .padding(start = 16.dp, bottom = 48.dp)
+                    .padding(start = 24.dp, bottom = 36.dp)
             ) {
-                Text(text = "$name, $age", color = Color.White, fontSize = 22.sp)
+                Text(text = "${profile.name}, ${profile.age}", color = Color.White, fontSize = 30.sp)
```

Comment:
Perhaps we could use TextStyles instead of specifying the size? `h1`/`p1` etc in the app in the future? It would make the design language more consistent.